.travis/pulp-smash-config.json

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
@pulp/qe guys, can you please take a look at qe related changes plus tests? thanks!